### PR TITLE
Fix resource_bundles misnaming

### PR DESCRIPTION
**Pull request by closure11**

### CODE REVIEW

File: ReachabilitySwift.podspec [hunk 1]
```diff
@@ -18,7 +18,7 @@ Pod::Spec.new do |s|
     :tag => 'v'+s.version.to_s
   }
   s.source_files = 'Sources/Reachability.swift'
-  s.resource_bundles = {"Kingfisher" => ["Sources/PrivacyInfo.xcprivacy"]}
+  s.resource_bundles = {"ReachabilitySwift" => ["Sources/PrivacyInfo.xcprivacy"]}
```
Comment (SethBaldwin) [inline ReachabilitySwift.podspec:21]: Does this need to match the `.name` or the `.module_name` property? 
Should it just be `{"Reachability" => ["Sources/PrivacyInfo.xcprivacy"]}`

Comment (tommyming) [reply ReachabilitySwift.podspec:21]: I think no matter how, the name should be renamed it back to the package.

**Comment by tommyming** [comment]
@ashleymills please review when you are free when you have time.
Sorry again for the mistakes in my previous PR.

File: ReachabilitySwift.podspec (hunk 1 above)
Comment (closure11) [reply ReachabilitySwift.podspec:21]: > Does this need to match the `.name` or the `.module_name` property? Should it just be `{"Reachability" => ["Sources/PrivacyInfo.xcprivacy"]}`

I tested both case with archiving an example project:

```
Example.xcarchive/Products/Applications/Example.app/Frameworks/Reachability.framework/Reachability.bundle/PrivacyInfo.xcprivacy

Example.xcarchive/Products/Applications/Example.app/Frameworks/Reachability.framework/ReachabilitySwift.bundle/PrivacyInfo.xcprivacy
```

Using XCode's Generate Privacy report feature, both cases generate the report correctly and include the library's privacy information. (Need to make some changes in reference to #406).

It doesn't look like we need to match any of them. More importantly, the existing `Kingfisher` naming would conflict with the real [Kingfisher](https://github.com/onevcat/Kingfisher) library if I also add that dependency:

image

So I think ReachabilitySwift would be a better choice, which avoids potential conflict issues with [tonymillion/Reachability](https://github.com/tonymillion/Reachability).

@SethBaldwin What do you think?

Comment (SethBaldwin) [reply ReachabilitySwift.podspec:21]: Yeah, that makes sense. Thanks for testing!

**Pull request merged**